packages/flutter/lib/src/cupertino/context_menu.dart

```diff
@@ -422,81 +433,79 @@ class _DecoyChild extends StatefulWidget {
 class _DecoyChildState extends State<_DecoyChild> with TickerProviderStateMixin {
   // TODO(justinmc): Dark mode support.
   // See https://github.com/flutter/flutter/issues/43211.
-  static const Color _lightModeMaskColor = Color(0xFF888888);
-  static const Color _masklessColor = Color(0xFFFFFFFF);
-
-  final GlobalKey _childGlobalKey = GlobalKey();
-  late Animation<Color> _mask;
   late Animation<Rect?> _rect;
+  late Animation<Decoration> _boxDecoration;
 
   @override
   void initState() {
     super.initState();
-    // Change the color of the child during the initial part of the decoy bounce
-    // animation. The interval was eyeballed from a physical iOS 13.1.2 device.
-    _mask = _OnOffAnimation<Color>(
-      controller: widget.controller,
-      onValue: _lightModeMaskColor,
-      offValue: _masklessColor,
-      intervalOn: 0.0,
-      intervalOff: 0.5,
-    );
-
-    final Rect midRect =  widget.beginRect!.deflate(
+    final Rect midRect = widget.beginRect!.deflate(
       widget.beginRect!.width * (_kOpenScale - 1.0) / 2,
     );
     _rect = TweenSequence<Rect?>(<TweenSequenceItem<Rect?>>[
       TweenSequenceItem<Rect?>(
         tween: RectTween(
           begin: widget.beginRect,
-          end: midRect,
-        ).chain(CurveTween(curve: Curves.easeInOutCubic)),
-        weight: 1.0,
+          end: widget.beginRect,
+        ).chain(CurveTween(curve: Curves.linear)),
+        weight: 3,
       ),
       TweenSequenceItem<Rect?>(
         tween: RectTween(
-          begin: midRect,
+          begin: widget.beginRect,
           end: widget.endRect,
-        ).chain(CurveTween(curve: Curves.easeOutCubic)),
-        weight: 1.0,
+        ).chain(CurveTween(curve: Curves.easeOutSine)),
+        weight: 5,
```

Comment:
Nit: Would it be useful to add a comment about these animations being eyeballed from iOS 14?  Or maybe that's already mentioned elsewhere.